operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 5.7.0-weekly.330
+version: 5.8.0-weekly.330
```

Comment:
Please, remove the changes around the version's bump. We typically don't tag a version on every change; instead, there is a dedicated CI job, that periodically releases a weekly version.

Reply:
Done.  Appreciate the pointer.